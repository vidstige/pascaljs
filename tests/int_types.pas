program inttypes;

var
    y: Byte;
    z: Word;
begin
    y := 4;
    z := 5;
    WriteLn('y: ', y, ', z: ', 5);
end.