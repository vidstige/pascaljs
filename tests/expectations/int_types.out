y: 4, z: 5
